homeassistant/components/immich/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: "No integration specific actions"
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: done
+    comment: "No integration specific actions"
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: done
+    comment: "No integration specific actions"
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: todo
+  discovery-update-info:
+    status: exempt
+    comment: "Service can't be discovered"
+  discovery:
+    status: exempt
+    comment: "Service can't be discovered"
```

Comment:
There's a custom addon we can use for discovery

Reply:
is a custom addon really something we need to consider here? Or is it more about the possibility that there could be a proper addon?

Reply:
This would also mean that you add the discovery part to the addon, Alex Belgium generally is open to adding those

Reply:
ok ... tbh I would like to let this for a later improvement and also keep it in state "exempt" for now, as this addon is no "core" addon and those the integration is not discover-able from a core point of view. Yes, it could be discover-able for some "foreign" addon, but then we've have to consider this for every core integration, which integrates an service, for which some custom addon could be out there